hrm-service/src/case/case-data-access.ts

```diff
@@ -7,6 +7,37 @@ import { DELETE_BY_ID } from './sql/case-delete-sql';
 import { selectSingleCaseByIdSql } from './sql/case-get-sql';
 import { Contact } from '../contact/contact-data-access';
 
+/**
```

Comment:
We would probably have to refactor our types a little if we were to go down this path - at least move them to other modules

Normally I'm of the opinion that types are best defined as close to the code that uses them as possible, but we definitely don't want HTTP related annotations polluting the data access modules, so we should probably move these types out to their own files in this case

TBF they started life as data access only types but are now used further up the stack, so there's probably no harm moving them out to a separate file anyway

Reply:
💯